strassengezwitscher/strassengezwitscher/models.py

```diff
@@ -1,7 +1,30 @@
 from __future__ import unicode_literals
-
+from decimal import Decimal
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
+from rest_framework.exceptions import ValidationError
+from rest_framework import filters
+
+
+class MapObjectFilter(filters.BaseFilterBackend):
```

Comment:
Man könnte überlegen, die Klasse in eine `filters.py` zu verschieben. Muss aber auch nicht zwingend.